### PR TITLE
Update ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by RithvikKasarla**
**Fixes issue:** #[Mention the issue number it fixes or add the details of the changes if it doesn't has a specific issue.]

**Changes:**
[Add here what changes were made in this pull request.]

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -8,7 +8,9 @@
 - [ ] Update to an existing algorithm
 - [ ] Error
 - [ ] Proposal to the Repository
-
+- [ ] Other
+** If Other Please Specify What you Did:**
```
Comment (arnavb) [inline .github/ISSUE_TEMPLATE.md]: Instead of writing:
** If Other Please Specify What you Did:**

Write:
** Specify What the Issue is:**

Comment (RithvikKasarla) [reply .github/ISSUE_TEMPLATE.md]: That makes more sense I could not figure out what to write

Comment (arnavb) [reply .github/ISSUE_TEMPLATE.md]: Please make the changes.

Comment (arnavb) [reply .github/ISSUE_TEMPLATE.md]: Please make the changes.

**Comment by arnavb** [comment]
@AdiChat could you make the changes I requested above (if they still apply) and then merge this?

**Comment by AdiChat** [comment]
The forked repository has been deleted, so, unfortunately, I cannot make the change.

**Comment by arnavb** [comment]
@AdiChat that's unfortunate; I think then this should be closed.

**Comment by AdiChat** [comment]
@RithvikKasarla We have closed this pull request but you can always open another pull request for any change anytime. :+1:

**Pull request closed without merging**